package.json

```diff
@@ -14,7 +14,7 @@
     "type": "git",
     "url": "git://github.com/mapbox/geojson-vt.git"
   },
-  "main": "src/index.js",
+  "main": "geojson-vt-dev.js",
```

Comment:
Because I'd like to have the bundled file ready for loading it up in a web browser. On second thought, we could use the `browser` field of `package.json` as [explained here](https://www.npmjs.com/package/browserify#browser-field).